### PR TITLE
Docs - Update UX issue templates

### PR DESCRIPTION
**Pull request by reitermb**
## Summary of Changes
Updates / Adds UX templates as proposed in https://hackmd.io/jxHOj3pETxCOD07tIckczg to issue templates repo folder 

## How to Test
Branch folder containing changes can be viewed here:
https://github.com/raft-tech/TANF-app/tree/docs/update-ux-templates/.github/ISSUE_TEMPLATE

The four UX templates are:
- https://github.com/raft-tech/TANF-app/blob/docs/update-ux-templates/.github/ISSUE_TEMPLATE/dev-ready-design-issue-template.md
- https://github.com/raft-tech/TANF-app/blob/docs/update-ux-templates/.github/ISSUE_TEMPLATE/research-facilitation-issue-template.md
- https://github.com/raft-tech/TANF-app/blob/docs/update-ux-templates/.github/ISSUE_TEMPLATE/research-planning-issue-template.md
- https://github.com/raft-tech/TANF-app/blob/docs/update-ux-templates/.github/ISSUE_TEMPLATE/research-synthesis-issue-template.md

### CODE REVIEW

**Review by stevenino — APPROVED** [review]
Looks good to me

File: .github/ISSUE_TEMPLATE/research-synthesis-issue-template.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+**AC:**
+
+- [ ] A hack.md with the drafted synthesis has been reviewed.
+- [ ] PR has been opened containing the final draft of the synthesis.
+
+**Tasks:**
+
+- [ ] --Aggregate notes from research sessions--
+
```
Comment (lfrohlich) [inline .github/ISSUE_TEMPLATE/research-synthesis-issue-template.md:22]:
```suggestion

- [ ] Document synthesis - Scope (research goal(s)), High-level findings, Actionable learnings, Issues that have to be added to Zenhub, Takeways for future research

```

Comment (lfrohlich) [inline .github/ISSUE_TEMPLATE/research-synthesis-issue-template.md:22]:
```suggestion
- [ ] Delete video recording(s) once notes are compiled
```

File: .github/ISSUE_TEMPLATE/research-facilitation-issue-template.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+---
+
+**Description:**
+
+*---Describe the planned timeline of research and link to research planning ticket. Should be pointed at 3 points.---*
+
+**AC:**
+
+- [ ] Research sessions have occurred.
+- [ ] Anonymized session notes or other top-line synthesis have been documented.
```
Comment (lfrohlich) [inline .github/ISSUE_TEMPLATE/research-facilitation-issue-template.md:17]: shouldn't top-line synthesis be part of synthesis issue?

File: .github/ISSUE_TEMPLATE/research-planning-issue-template.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+*---Describe the scope of research and core questions/designs to be investigated. Reference any related tickets that might be helpful to cite for planning, discussion, or review. Should typically be between 5 and 13 points.---*
+
+**AC:**
+- [ ] A research plan has been created which includes:
+    - [ ] Relevant research questions.
+    - [ ] Recruitment criteria & target participants.
+    - [ ] Recruitment/participation email templates.
+    - [ ] Timeline (for recruitment and for research sessions).
+    - [ ] Testable Designs/Content (if applicable).
+
```
Comment (lfrohlich) [inline .github/ISSUE_TEMPLATE/research-planning-issue-template.md:21]:
```suggestion

  - [ ] Ethics, privacy concerns 

```

File: .github/ISSUE_TEMPLATE/research-planning-issue-template.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+---
+
+**Description:**
+
+*---Describe the scope of research and core questions/designs to be investigated. Reference any related tickets that might be helpful to cite for planning, discussion, or review. Should typically be between 5 and 13 points.---*
+
+**AC:**
+- [ ] A research plan has been created which includes:
+    - [ ] Relevant research questions.
```
Comment (lfrohlich) [inline .github/ISSUE_TEMPLATE/research-planning-issue-template.md:16]:
```suggestion
    - [ ] Research goals, relevant research questions.
```

**Review by lfrohlich — COMMENTED** [review]
Added some comments throughout

File: .github/ISSUE_TEMPLATE/dev-ready-design-issue-template.md [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+—As a [user type] I want the page to exist so that I can see it.—
+
+**Recommended pa11y actions:**
+
+—‘click element #error-report’,—
+
+**AC:**
+
+- [ ] Documentation work for the following has occurred:
+   - [ ] Relevant User stories.
```
[the remark this answers is not in the document]
Comment (reitermb) [reply .github/ISSUE_TEMPLATE/dev-ready-design-issue-template.md:25]: These are just ensuring that these dev-ready design tickets remain refined and capture the final set of high level user stories. Similar to how we've been updating the V2 notifications content ticket to stay up to date with the doc & the version slicing decisions in there.

**Review by lfrohlich — APPROVED** [review]
LGTM!

**Pull request merged**